### ISSUE TITLE
Autopilot much slower than expected

### ISSUE THREAD

**Issue by DynV** [pioneerspacesim/pioneer#5548]
### Observed behaviour
As I was finished my scanning of an ~750Km r asteroid I had trouble with my auto-pilot being set on my next target, a ~500Km r asteroid, my ship moved erratically. I moved manually toward my target ~1 min then set my auto-pilot again, it was bobbing somewhat hard toward my target then the direction became stable but the speed was clearly less than what I expect it would be for such a trip. I moved further from the origin then made the attached save.

Setting the auto-pilot on the target again get stuck at a speed considerably lower than what I expect. It seems to hover very close to 1Km/s from the origin (the gray target, the strongest gravitation) and the relatively slow speed toward the target slowly drop; ~3Km/s while I need to go all the way to 72Km/s to reach a green bar (on the right). I have a feeling the auto-pilot tethers my ship to the origin and I'm going toward my target the most the tether allow.

### Expected behaviour
Have a speed toward the target the fastest it can until it reaches ~3/4 of what I'd get from a green bar (on the right).

### Steps to reproduce
1. Load attached gamesave
2. turn on autopilot to low orbit of the target already set
3. pay attention to the speed of the origin (in grey on the left).

__My pioneer version (and OS):__
20230203-win

The attached file was simply renamed so it would go through the upload extension filter, there was no modification to the file content of my save-file.
[Pioneer-Yoan_Faure-20230203-autopilot_tethered_on_Rhea.zip](https://github.com/pioneerspacesim/pioneer/files/10765862/Pioneer-Yoan_Faure-20230203-autopilot_tethered_on_Rhea.zip)

**Comment by Web-eWorks** [comment]
CC @azieba for general autopilot issues (may be an issue with targeting a faraway body with high orbit speed).

**Comment by DynV** [comment]
Another time but unfortunately my start was over the horizon so my speed is away from the target (but still very close to 1Km/s from the origin).
[Pioneer-Yoan_Faure-20230311-autopilot_tethered_on_Phoebe.zip](https://github.com/pioneerspacesim/pioneer/files/10773671/Pioneer-Yoan_Faure-20230311-autopilot_tethered_on_Phoebe.zip)

**Comment by azieba** [comment]
@DynV I'm having trouble loading your save files in version 20230203. Are you sure this is your version?

Fatal: [T] libs/CargoManager.lua:64: attempt to index field '?' (a nil value)

**Comment by nozmajner** [comment]
I also had that problem with the save. Do you have any mods active?

**Comment by Gliese852** [comment]
Indeed, it does not load, although the save version is 89 (introduced in latest release)

**Comment by DynV** [comment]
See for yourselves:

> C:\Users\[user name]>dir %UserProfile%\OneDrive\Documents\Pioneer\mods
>  Volume in drive C has no label.
>  Volume Serial Number is F848-5889
> 
>  Directory of C:\Users\[user name]\OneDrive\Documents\Pioneer\mods
> 
> 02/12/2023  04:09 PM    <DIR>          .
> 02/12/2023  04:09 PM    <DIR>          ..
>                0 File(s)              0 bytes
>                2 Dir(s)  24,536,346,624 bytes free

![screenshot-20230218-145316](https://user-images.githubusercontent.com/2823807/219890045-d959e298-1b50-497e-9817-466258443850.png)

**Comment by mkochgdn3810** [comment]
There are a couple more issues with Rhea. I do not have a proof they 100% related to the original reported issue, but I believe so, so here a some saves and footage of them.
Is you interested in only save file and footage directly related to this issue - take a look at the attachments with name `3_SlowSpeed_241c3194b_Manjaro`.
I have a 1920x1080 footage as well, but it weights too much for GitHub.

[README.md](https://github.com/pioneerspacesim/pioneer/files/14675862/README.md)

https://github.com/pioneerspacesim/pioneer/assets/106257000/01bcb1dd-c0dd-497b-b216-4ca10e3082e9

[4_RheaArtifactsMore_241c3194b_Manjaro.zip](https://github.com/pioneerspacesim/pioneer/files/14675863/4_RheaArtifactsMore_241c3194b_Manjaro.zip)

https://github.com/pioneerspacesim/pioneer/assets/106257000/562dc75f-40a6-4560-9392-fb3a70571ddc

[4_RheaArtifacts_241c3194b_Manjaro.zip](https://github.com/pioneerspacesim/pioneer/files/14675864/4_RheaArtifacts_241c3194b_Manjaro.zip)

https://github.com/pioneerspacesim/pioneer/assets/106257000/2e8ae705-1d09-479e-8897-dd731110d162

[3_SlowSpeed_241c3194b_Manjaro.zip](https://github.com/pioneerspacesim/pioneer/files/14675865/3_SlowSpeed_241c3194b_Manjaro.zip)

https://github.com/pioneerspacesim/pioneer/assets/106257000/2371df03-f589-418c-8c17-b17f81d7585d


https://github.com/pioneerspacesim/pioneer/assets/106257000/1246a0a7-5ccd-49ff-ab4f-c60646492595


https://github.com/pioneerspacesim/pioneer/assets/106257000/47d4bfd8-3697-4d9d-bb8c-7e1fb44331a0

[2_RegainControls_241c3194b_Manjaro.zip](https://github.com/pioneerspacesim/pioneer/files/14675866/2_RegainControls_241c3194b_Manjaro.zip)
[1_ToRhea_241c3194b_Manjaro.zip](https://github.com/pioneerspacesim/pioneer/files/14675867/1_ToRhea_241c3194b_Manjaro.zip)

https://github.com/pioneerspacesim/pioneer/assets/106257000/a3302c82-4ab9-4d3c-8e41-a0e556109bc8
